terraform.tfstate

```diff
@@ -0,0 +1,8 @@
+{
+  "version": 4,
+  "terraform_version": "1.0.11",
+  "serial": 1,
+  "lineage": "0b71013c-222b-8abb-ec3a-b02e441813d2",
+  "outputs": {},
+  "resources": []
+}
```

Comment:
This file should not be committed, please remove it

Reply:
removed it as per request